### PR TITLE
blog: announcement for JUnit Rules - Arquillian Core 1.2.0.Final

### PR DESCRIPTION
**Pull request by bartoszmajsak**
Copy of #415 to generate blog preview

### CODE REVIEW

File: blog/arquillian-core-1.2.0.Final.textile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+date: 2017-12-20
+author: dipak-pawar
+title: Welcome JUnit Rules for Arquillian
```
Comment (bartoszmajsak) [inline blog/arquillian-core-1.2.0.Final.textile]: Have a look at other release posts - I think only `date` and `author` are necessary.

Title can be wrapped in `h3.` instead

File: blog/arquillian-core-1.2.0.Final.textile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+We have released yet another version of Arquillian Core. This time with one new feature.
```
Comment (bartoszmajsak) [inline blog/arquillian-core-1.2.0.Final.textile]: I would drop this paragraph

File: blog/arquillian-core-1.2.0.Final.textile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+h3. What's new?
```
Comment (bartoszmajsak) [inline blog/arquillian-core-1.2.0.Final.textile]: Use of the blog instead

File: blog/arquillian-core-1.2.0.Final.textile [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ Greeter greeter;
+
+ @Test
+ public void should_greet_earthlings() throws Exception {
+     String name = "Earthlings";
+     Assert.assertEquals("Hello, " + name, greeter.greet(name));
+ }
+
+p. In order to use any other JUnit Class or Method Rule with @ArquillianClassTest@ or @ArquillianTest@ rule, you have to use it with @RuleChain@ having new methods defined as outer rules. e.g.
+
```
Comment (bartoszmajsak) [inline blog/arquillian-core-1.2.0.Final.textile]: Are these really new methods or new rules?

File: blog/arquillian-core-1.2.0.Final.textile [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+     String name = "Earthlings";
+     Assert.assertEquals("Hello, " + name, greeter.greet(name));
+ }
+
+p. In order to use any other JUnit Class or Method Rule with @ArquillianClassTest@ or @ArquillianTest@ rule, you have to use it with @RuleChain@ having new methods defined as outer rules. e.g.
+  
+bc(prettify).. @Rule
+public MethodRule testWatchman = MethodRuleChain.outer(new ArquillianTest().around(new TestWatchman());
+
+p. Arquillian is not enforcing user to define both rules as implementations of @MethodRuleChain@ is different for different users. JUnit doesn't provide @MethodRuleChain@ similar to the @RuleChain@, as we don't know your implementation of @MethodRuleChain@, we can't check if your MethodRuleChain has @ArquillianTest@ rule defined.
```
Comment (bartoszmajsak) [inline blog/arquillian-core-1.2.0.Final.textile]: I don't really know what you wanted to say here. It's a bit confusing for me. Maybe just worth mentioning that there is no chain for method rules out of the box provided by JUnit (linking to the issue) so we have our own and you can use it if you want

**Comment by alien-ike** [comment]
### Blog Preview
[Build 203](https://travis-ci.org/arquillian/arquillian.github.io/builds/320100958)

blog-preview

File: blog/arquillian-core-1.2.0.Final.textile (hunk 4 above)
Comment (dipak-pawar) [reply blog/arquillian-core-1.2.0.Final.textile]: > having new methods defined as outer rules.

having arquillian rules defined as outer rules.

File: blog/arquillian-core-1.2.0.Final.textile (hunk 5 above)
Comment (dipak-pawar) [reply blog/arquillian-core-1.2.0.Final.textile]: I am fine with it.
> There is no chain for method rules out of the box provided by JUnit (https://github.com/junit-team/junit4/issues/383#issuecomment-5056440) so we have our own and you can use it if you want

**Comment by alien-ike** [comment]
### Blog Preview
[Build 209](https://travis-ci.org/arquillian/arquillian.github.io/builds/320221410)

blog-preview

**Comment by alien-ike** [comment]
### Blog Preview
[Build 211](https://travis-ci.org/arquillian/arquillian.github.io/builds/320235910)

blog-preview

**Comment by alien-ike** [comment]
### Blog Preview
[Build 213](https://travis-ci.org/arquillian/arquillian.github.io/builds/320239942)

blog-preview

**Pull request merged**